unvtray/assets/net.unvanquished.unvtray.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Version=1.5
+Name=Unvanquished Tray
+Comment=Minimalistic Unvanquished server browser that runs in your system tray
+Icon=unvtray
+Terminal=false
+Type=Application
+Exec="unvtray"
```

Comment:
Just a remark/no change request: For pip installations, it would be more robust to call `python -m unvtray` as this works regardless of whether `~/.local/bin` is in `PATH`. But since we can only package `.desktop` with non-pip distributions (see PR comment), this is fine/better as the launcher script will end up in `/usr/bin`.

Reply:
Right, that `.desktop` file actually expects there is a `unvtray` program in `$PATH`. It would probably easier to add a simple script doing `python -m unvtray` if a systems lacks such program, than doing it the other way and expect all systems to provide a working `python -m unvtray`.